src/UI/Internal/Text.elm

```diff
@@ -276,3 +262,29 @@ mobileAttributes size =
             , Font.letterSpacing 2
             , Font.extraBold
             ]
+
+
+mapOpt : (Options -> Options) -> Text -> Text
```

Comment:
I would call this `mapOptions` or `mapOpts` as this is not mapping an `Option`, this mapping `Options`.